notebooks/src/3-create-stac-item-with-extension.ipynb

```diff
@@ -425,7 +1838,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": 26,
    "metadata": {},
    "outputs": [],
    "source": [
```

Comment:
for markdown cell below, change to "Now you have created an Item with a few popular STAC Extensions. In the [following tutorial](/en/tutorials/4-create-stac-collection/index.html), you will learn how to add a Collection to this STAC Catalog."